com.sap.cloud.lm.sl.cf.api/src/main/resources/mtarest.yaml

```diff
@@ -0,0 +1,588 @@
+---
```

Comment:
It is an autogenerated file after each build - I was thinking of persisting it in the repo since that way it will be clear when a change has been made to either of the swagger files - and so the developer would be reminded to upload the changed files to the swagger site.